cmake/simdjson-user-cmakecache.cmake

```diff
@@ -1,24 +1,19 @@
 #
-# ${SIMDJSON_USER_CMAKECACHE} contains the *user-specified* simdjson options so you can call cmake on
-# another branch or repository with the same options.
+# ${USER_CMAKECACHE} contains the *user-specified* simdjson options so you can
```

Comment:
What's the thought behind removing SIMDJSON_? Is this a standard variable?

Reply:
I turned this into a script that is called with `-P` mode, it's the first custom target after the developer mode delimiter in the root CML.